packages/react-jsx-highcharts/package.json

```diff
@@ -72,6 +71,10 @@
     "prop-types": "^15.0.0"
   },
   "browserslist": [
-    "ie >= 11"
+    "chrome >= 81",
+    "edge >= 81",
+    "firefox >= 78",
+    "ios >= 12",
+    "safari >= 12"
```

Comment:
Might be able to replace this with something like `'since 2020'`

Reply:
The ["since 2020"](https://browserslist.dev/?q=c2luY2UgMjAyMA%3D%3D) picks up QQ Browser, which I don't know at all.

The "since 2020" browsers can also change later on to include an unknown browser. I kind of like to be explicit here.

The next react-router has also defined the browsers the same way: https://github.com/remix-run/react-router/blob/main/.browserslistrc